src/tests/_internal/test_models.py

```diff
@@ -18,6 +25,13 @@
         pytest.param(1, None, "tpu-v3", "google", "v3", id="one-tpu-vendor-not-set"),
         pytest.param(1, "google", "tpu-v5p", "google", "v5p", id="one-tpu-vendor-is-set"),
         pytest.param(1, AMD, "MI300X", "amd", "MI300X", id="cast-enum-to-string"),
+        pytest.param(1, AMD, "MI300A", "amd", "MI300A", id="cast-enum-to-string"),
```

Comment:
These new test cases seem redundant, they all check (enum) AMD to (string) "amd" casting.

Reply:
Yea, I realised this myself in the evening 👍